### PR TITLE
Adds a PR template

### PR DESCRIPTION
**Pull request by abea**
PR template meant primarily to help us remember to update the changelog and keep changes manageable.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Please ensure your pull request follows these guidelines:
+
+- [ ] Link to the related issue with requirements and/or clearly describe 1) what problem this solves and 2) the requirements to review it against.
+- [ ] Update the `CHANGELOG.md` file with the added feature or fix. If there is not yet a heading for an upcoming release, add one following the established pattern.
+- [ ] Keep the pull request minimal! Break large updates into separate pull requests whenever possible. Small requests get reviewed more quickly.
```
Comment (boutell) [inline .github/pull_request_template.md:5]: As long as they make sense independently and can be tested independently.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (boutell) [inline .github/pull_request_template.md:6]: - [ ] Follow our style guide. Most modules have an eslint configuration, so you can run `npx eslint .` in the module if your editor does not automatically pick up on that.

**Review by boutell — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -2,6 +2,7 @@ Please ensure your pull request follows these guidelines:
 
 - [ ] Link to the related issue with requirements and/or clearly describe 1) what problem this solves and 2) the requirements to review it against.
 - [ ] Update the `CHANGELOG.md` file with the added feature or fix. If there is not yet a heading for an upcoming release, add one following the established pattern.
-- [ ] Keep the pull request minimal! Break large updates into separate pull requests whenever possible. Small requests get reviewed more quickly.
+- [ ] Keep the pull request minimal! Break large updates into separate pull requests for individual features/fixes whenever possible. Small requests get reviewed more quickly.
+- [ ] All tests must pass, including the linters. Run `npm run lint` to test code linting independently.
```
Comment (boutell) [inline .github/pull_request_template.md:6]: If the module does not yet have a `lint` task please adhere to the formatting style in place.

**Review by boutell — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 3 above)
Comment (abea) [reply .github/pull_request_template.md:6]: @boutell This is a template only for core, which has the test.

**Review by boutell — APPROVED** [review]

**Pull request merged**